### DISCUSSION TITLE
Expo support for React Native Reanimated 3

### DISCUSSION THREAD

**Discussion by ponikar** [software-mansion/react-native-reanimated#4148] [Q&A]
When I install the package via `expo install react-native-reanimated` It is still installing version `2.x.x` 

So I want to know how to use the latest version of the expo. If I install it with `yarn` or `npm` The app is simply crashing on runtime.

**Comment by chiqors** [comment]
i would like to know about this too

**Comment by Nantris** [comment]
I'm also curious to know about this since upgrading to SDK 48 has suddenly begun triggering: `Animated node with id [3] already exists` - but we only have the one version of `react-native-reanimated`
